week2/homework/src/commands.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+const colors = require('colors');
```

Comment:
First Error appears when I launched the application was `module not found`

when you are adding a new module using `npm` make sure to use `--save` option to save it as a dependency that will make any one using your program prepare it with a single command `npm install` not warring about what modules he have to include, what if you have around 10 to 20 module and each should be in it's specific version to run as it's right way?

Reply:
Just to clarify, inside src folder I have, addToDos, commands.js, index.js, removeList.js, resetList.js and showList.js files. Now I need inside src folder run `npm init` right?

Reply:
you have a related `package.json` exist already with the main key it have, so you may just have to install the packages there.

once you have that done, who ever is using the program will be notified to install any packages with `npm install` command only once he have your project from reading the `package.json` file.